app/renderer/components/main/braveryPanel.js

```diff
@@ -242,7 +242,7 @@ class BraveryPanel extends ImmutableComponent {
     })
     const compactBraveryPanel = getSetting(settings.COMPACT_BRAVERY_PANEL)
 
-    return <Dialog onHide={this.props.onHide} testId='braveryPanelContainer' isClickDismiss>
+    return <Dialog onHide={this.props.onHide} testId='braveryPanelContainer' isClickDismiss isBraveryPanel>
```

Comment:
small: let's place `isClickDismiss`, `isBraveryPanel` and other props that are `true` by default in the beginning of component's props. No need to do this now since this component is small enough but please do for future changes. that helps readability and even more for us that do front-end a lot. 

I'm going to update our styles doc soon with that info btw